wagtailio/project_styleguide/templates/patterns/pages/feature_index_page/feature_index_page.html

```diff
@@ -0,0 +1,12 @@
+{% extends "patterns/base_page.html" %}
+{% load wagtailcore_tags wagtailimages_tags static %}
+
+{% block content %}
+    {% if page.subheading %}<h3>{{ page.subheading }}</h3>{% endif %}
+    {% for block in page.features %} ☐ {{ block.value.heading }} {% endfor %}
```

Comment:
Note: Could this not be done with the features template?

Reply:
Thanks for the guidance @William-Blackie! I've made some revisions in 7a147be and 35d6274